src/FluentCMS.Web.UI.Components/Components/Toast/ToastProvider.razor

```diff
@@ -0,0 +1,53 @@
+﻿@inherits BaseComponent
+
+@namespace FluentCMS.Web.UI.Components
+
+<div @attributes="AdditionalAttributes"
+     class="@this.GetClasses()">
+    @foreach (var toast in toasts)
+    {
+        <Toast Dismissible="true"
+               Type="toast.Type">
+            @toast.Message
+        </Toast>
+    }
+</div>
+
+@code
+{
+    [Inject]
+    ToastService toastService { get; set; } = default!;
+
+    List<ToastModel> toasts = new List<ToastModel>();
+
+    public void Show(string message, ToastType? type)
+    {
+        var toast = new ToastModel()
+            {
+                Duration = 5000,
+                Message = message,
+                Type = type ?? ToastType.Default
+            };
+
+        toasts.Add(toast);
+
+        InvokeAsync(StateHasChanged);
+
+        Timer? timer = null;
```

Comment:
Timer needs to be global not local Garbage collector can dispose it before the callback in local timers.